common/python/schema_validation/data/WorkOrderSubmit.json

```diff
@@ -0,0 +1,169 @@
+        "pattern": "^(0[x|X])?[0-9a-fA-F]+$",
+        "error_msg":
+        "Invalid data format for verifyingKey"},
+    "inData": {
+        "type": "array",
+        "uniqueItems": true,
+        "additionalProperties": false,
+        "items": {
+        "type": "object",
+        "error_msg":
+        "Invalid data format for inData ",
+        "properties": {
+            "index": {
+                "type": "integer",
+                "error_msg":
+                "Invalid data format for inData index"},
+            "data": {
+                "type": "string",
+                "pattern": "[0-9A-Za-z]",
+                "error_msg":
+                "Invalid data format for in data"},
+            "dataHash": {
+                "type": "string",
+                "pattern": "^(0[x|X])?[0-9a-fA-F]+$",
+                "error_msg":
+                "Invalid data format for data hash of in data"},
+            "encryptedDataEncryptionKey": {
+                "type": "string",
+                "pattern": "^-$|^$|^null$|^(0[x|X])?[0-9a-fA-F]+$",
+                "error_msg":
+                "Invalid data format for Encryption key of in data"},
+            "iv": {
+                "type": "string",
+                "pattern": "^$|^(0[x|X])?[0-9a-fA-F]+$",
+                "error_msg":
+                "Invalid data format for initialization vector of in data"}
+        },
+        "required": ["index", "data"]
+        }
+    },
+    "outData": {
+        "type": "array",
+        "uniqueItems": true,
+        "additionalProperties": false,
+        "items": {
+        "type": "object",
+        "error_msg":
+        "Invalid data format for outData ",
+        "properties": {
+            "index": {
+                "type": "integer",
+                "error_msg":
+                "Invalid data format for outData index"},
+            "dataHash": {
+                "type": "string",
+                "pattern": "^(0[x|X])?[0-9a-fA-F]+$",
+                "error_msg":
+                "Invalid data format for data hash of out data"},
+            "data": {
+                "type": "string",
+                "pattern": "[0-9A-Za-z]",
+                "error_msg":
+                "Invalid data format for out data"},
+            "encryptedDataEncryptionKey": {
+                "type": "string",
+                "pattern": "^-$|^$|^null$|^(0[x|X])?[0-9a-fA-F]+$",
+                "error_msg":
+                "Invalid data format for Encryption key of out data"},
+            "iv": {
+                "type": "string",
+                "pattern": "^$|^(0[x|X])?[0-9a-fA-F]+$",
+                "error_msg":
+                "Invalid data format for initialization vector of out data"}
+        }
+        }
+    }
+},
+"required": [
+    "responseTimeoutMSecs",
+    "payloadFormat",
+    "workOrderId",
+    "workerId",
+    "workloadId",
+    "requesterId",
+    "encryptedSessionKey",
+    "requesterNonce",
+    "encryptedRequestHash",
+    "inData"
```

Comment:
inData is not a mandatory field. We can submit work order request without inData. For example if you want a sample workload with does some fixed calculation we don't need inData

Reply:
done